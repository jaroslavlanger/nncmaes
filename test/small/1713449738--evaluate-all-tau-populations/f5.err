2024-04-18 13:02:38.485732: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-04-18 13:02:38.485922: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-04-18 13:02:38.488952: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-04-18 13:02:38.510354: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-18 13:02:43.228484: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
2024-04-18 13:02:50.606793: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:388] MLIR V1 optimization pass is not enabled
2024-04-18 13:02:50.664739: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:02:51.712765: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:02:53.137727: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:02:54.503165: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:02:55.894448: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:07.517137: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:08.790735: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:10.298419: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:12.104426: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:13.751702: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:27.100538: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:28.325464: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:29.769588: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:31.588463: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:34.119230: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:54.145519: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:55.739376: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:57.479684: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:03:59.358852: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:04:01.118751: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:04:23.682506: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:04:26.088070: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:04:28.779180: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:04:31.294496: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:04:33.960299: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:04:59.502783: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:01.937855: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:05.558748: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:07.934631: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:11.932947: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:49.076279: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:51.945929: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:54.600295: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:05:58.562434: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:06:01.557307: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:06:29.708230: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:06:32.279916: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:06:35.923909: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:06:39.137935: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:06:43.821371: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:07:17.072870: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:07:19.889752: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:07:25.656270: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:07:29.671382: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:07:33.008842: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:08:06.430013: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_1/kernel/Adam_1/Assign' id:150 op device:{requested: '', assigned: ''} def:{{{node dense_1/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_1/kernel/Adam_1, dense_1/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:08:08.558803: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_3/kernel/Adam_1/Assign' id:545 op device:{requested: '', assigned: ''} def:{{{node dense_3/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_3/kernel/Adam_1, dense_3/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:08:11.347731: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_5/kernel/Adam_1/Assign' id:1008 op device:{requested: '', assigned: ''} def:{{{node dense_5/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_5/kernel/Adam_1, dense_5/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:08:17.494833: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_7/kernel/Adam_1/Assign' id:1474 op device:{requested: '', assigned: ''} def:{{{node dense_7/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_7/kernel/Adam_1, dense_7/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
2024-04-18 13:08:22.345074: W tensorflow/c/c_api.cc:305] Operation '{name:'dense_9/kernel/Adam_1/Assign' id:1907 op device:{requested: '', assigned: ''} def:{{{node dense_9/kernel/Adam_1/Assign}} = AssignVariableOp[_has_manual_control_dependencies=true, dtype=DT_DOUBLE, validate_shape=false](dense_9/kernel/Adam_1, dense_9/kernel/Adam_1/Initializer/zeros)}}' was changed by setting attribute after it was run by a session. This mutation will have no effect, and will trigger an error in the future. Either don't modify nodes after running them or create a new session.
