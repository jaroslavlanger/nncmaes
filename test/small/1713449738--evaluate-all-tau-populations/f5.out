(3_w,6)-aCMA-ES (mu_w=2.0,w_1=63%) in dimension 2 (seed=1, Thu Apr 18 13:02:49 2024)
tau= 0.60
tau= 0.60
tau= 0.60
tau= 0.20
tau= 1.00
tau= 0.07
tau= 0.47
tau= 0.60
tau= 0.60
tau=  nan
 2D  5-fun   1-inst  500-budget  True-solved  63-evals   60-cma-evals  374s 0-restarts  6-init-pop (Surrogate(model=Raf(data_noise=0.01), subset=ClosestToEachTestPoint(Mahalanobis, n_max_coef=20), sns=shift_and_scale_x_by_es_y_by_train), evaluate_all_kendall, n_min_fn=get_dim) 0.0e+00-delta-f          1-seed
